docs/reference/content/whats-new.md

```diff
@@ -63,6 +63,48 @@ Support for `Instant`, `LocalDate` and `LocalDateTime` has been added to the dri
 and / or more flexibility regarding JSR-310 dates should check out the alternative JSR-310 codecs provider by Cezary:
 [bson-codecs-jsr310](https://github.com/cbartosiak/bson-codecs-jsr310).
 
+### JSR-305 NonNull annotations
+
+The public API is now annotated with JSR-305 compatible @NonNull and @Nullable annotations.  This will allow programmers
+to rely on tools like FindBugs/SpotBugs, IDEs like IntelliJ IDEA, and compilers like the Kotlin compiler to find errors in the use of the 
+driver via static analysis rather than via runtime failures, which may happen infrequently.
```

Comment:
This is hard to parse - suggest removing the last part ', which may happen infrequently.'

Reply:
Done